examples/usercall-extension/app/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "app"
+version = "0.1.0"
+authors = ["Vardhan Thigle <vardhan.thigle@fortanix.com>"]
+
+#[profile.dev]
+panic = "abort"
```

Comment:
What are you trying to do here?

Reply:
This was, since I was trying to use gdb to debug the panic. Panic does not happen on latest nightly.